app/assets/stylesheets/custom.scss

```
@@ -71,3 +71,50 @@ p {
   padding-top: 9px;
   font-weight: bold;
 }
+
+/* microposts */
+.microposts {
```

CSSだとこう書くイメージがある。
```suggestion
.micropost-list {
```

FYI)
[CSSのクラス名を決めるときに使うリストをつくりました \#CSS \- Qiita](https://qiita.com/manabuyasuda/items/dbb76ed36970bec95470)